Makefile

```diff
@@ -47,6 +47,7 @@ $(MODULE_ROOTS):
 	@echo "generate k8s object for module $(notdir $@)"
 	@pushd $@ 1>/dev/null 2>&1; \
 	controller-gen object:headerFile="$(GO_HEADER_FILE)" paths="./api/..."; \
+	controller-gen crd paths="./api/..." output:crd:dir="./charts/crds"; \
```

Comment:
these two commits seem to can be squashed

Reply:
update